game/scripts/npc/abilities/dazzle_shallow_grave.txt

```diff
@@ -26,7 +26,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "24 22 20 18 17 16" //OAA
+    "AbilityCooldown"                                     "24 22 20 18 18 18" //OAA
```

Comment:
I calculated that with Octarine Core, Aghanim Scepter and Spell Prism and if Dazzle spams every spell off cd he can get Permanent Grave. Even if Dazzle can't use Poison Touch and scepter ability consistently he can achieve this (there is 0.88s time of no Grave). So I would nerf this to 24 seconds at all levels.